### PR TITLE
Update doorbird.markdown

### PR DESCRIPTION
**Pull request by rosscullen**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

Additional tweaks to make easier to understand. Automation examples. Addition of Doorbird Web Portal use for easier configuration.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/doorbird.markdown [hunk 1]
```diff
@@ -80,7 +82,7 @@ The URLs on the event will be based on the configuration used to connect to your
 
 #### Schedules
 
-Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported Doorbird devices).
+Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS or the web portal at https://webadmin.doorbird.com. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported Doorbird devices). Essentially, you can enable a HTTP(S) call from your Doorbird device to the Home Assistant Doorbird API by configuring an action/event (by enabling a schedule).
```
Comment (frenck) [inline source/_integrations/doorbird.markdown]: Please do not use bare URLs.

File: source/_integrations/doorbird.markdown [hunk 2]
```diff
@@ -25,11 +25,13 @@ The `doorbird` implementation allows you to integrate your [DoorBird](https://ww
 There is currently support for the following device types within Home Assistant:
 
 - [Camera](#camera) - View live and historical event based images.
-- [Button](#button) - Enable control of relays and camera night vision.
+- [Button](#button) - Enable control of relays and camera low-light/night vision (IR).
 
 ## Setup
 
-It is recommended to set up a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:
+It is recommended to set up a new & dedicated account on your Doorbird App/web portal for use with Home Assistant. The instructions in this documentation refer specifically to the Doorbird IOS/Android app but most of the instructions can also be performed using the Doorbird web portal at https://webadmin.doorbird.com and logging in on your Doorbird admin account.
```
Comment (frenck) [inline source/_integrations/doorbird.markdown]: Please do not use bare URLs

**Review by frenck — CHANGES_REQUESTED** [review]
Please avoid the use of bare URLs, additionally, please make sure the build passes. Thanks! 👍 

../Frenck

**Review by rosscullen — COMMENTED** [review]
Minor changes to doorbird Spelling (DoorBird)

**Review by rosscullen — COMMENTED** [review]
minor spelling changes to docs

File: source/_integrations/doorbird.markdown [hunk 3]
```diff
@@ -94,10 +96,12 @@ Click on the dropdown button in the top left and choose your specific "Schedule
 
 On the desired event, you should be able to specify blocks of time for when you would like the event to be sent to Home Assistant. If you want the event to always send, the square in the upper right can be used to populate the entire schedule. Events will be fired to Home Assistant for blocks of time that are blue.
 
-Note: Remember to complete the schedule assignment steps above for each event type that you registered.
+Note: Remember to complete the schedule assignment steps above for each event type that you registered. If you are configuring HTTP Call(s) schedules for registered RFID tags, please note you will have to enable/configure schedule for each individual RFID tag.
```
Comment (bdraco) [inline source/_integrations/doorbird.markdown]:
```suggestion
Note: Remember to complete the schedule assignment steps above for each event type you registered. If you are configuring HTTP Call(s) schedules for registered RFID tags, please note that you will have to enable/configure the schedule for each RFID tag.
```

File: source/_integrations/doorbird.markdown [hunk 4]
```diff
@@ -75,14 +77,14 @@ The following keys are available on `event_data`:
...
 
 <div class="note">
-The URLs on the event will be based on the configuration used to connect to your Doorbird device. Ability to connect from outside your network will depend on your configuration.
+The URLs on the event will be based on the configuration used to connect to your DoorBird device. Ability to connect from outside your network will depend on your configuration.
 </div>
 
 #### Schedules
 
-Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported Doorbird devices).
+Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS or the [DoorBird - WebAdmin](https://webadmin.doorbird.com) portal. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported DoorBird devices). Essentially, you can enable a HTTP(S) call from your DoorBird device to the Home Assistant DoorBird API by configuring an action/event (by enabling a schedule).
```
Comment (bdraco) [inline source/_integrations/doorbird.markdown]:
```suggestion
Once events have been registered on the DoorBird device, they must be attached to a schedule using the official DoorBird app on Android or iOS or the [DoorBird - WebAdmin](https://webadmin.doorbird.com) portal. Currently, there are schedules available for doorbell, motion, relay, and RFID events (on supported DoorBird devices). Essentially, you can enable an HTTP(S) call from your DoorBird device to the Home Assistant DoorBird API by configuring an action/event (by enabling a schedule).
```

File: source/_integrations/doorbird.markdown [hunk 5]
```diff
@@ -75,14 +77,14 @@ The following keys are available on `event_data`:
 - `html5_viewer_url`
 
 <div class="note">
-The URLs on the event will be based on the configuration used to connect to your Doorbird device. Ability to connect from outside your network will depend on your configuration.
+The URLs on the event will be based on the configuration used to connect to your DoorBird device. Ability to connect from outside your network will depend on your configuration.
```
Comment (bdraco) [inline source/_integrations/doorbird.markdown]:
```suggestion
The URLs on the event will be based on the configuration used to connect to your DoorBird device. The ability to connect from outside your network will depend on your configuration.
```

File: source/_integrations/doorbird.markdown [hunk 6]
```diff
@@ -64,7 +66,7 @@ If DoorBird was setup using UI prompts, a token can be discovered through DoorBi
 
 #### Event Data
 
-Each event will include live image and video URLs for the Doorbird device that triggered the event. These URLs can be found on the event data and can be useful in automation actions. For example, you could use `html5_viewer_url` on a notification to be linked directly to the live view of the device that triggered the automation.
+Each event will include live image and video URLs for the DoorBird device that triggered the event. These URLs can be found on the event data and can be useful in automation actions. For example, you could use `html5_viewer_url` on a notification to be linked directly to the live view of the device that triggered the automation.
```
Comment (bdraco) [inline source/_integrations/doorbird.markdown]:
```suggestion
Each event will include live image and video URLs for the DoorBird device that triggered the event. These URLs can be found on the event data and are helpful in automation actions. For example, you could use `html5_viewer_url` on a notification to be linked directly to the live view of the device that triggered the automation.
```

File: source/_integrations/doorbird.markdown [hunk 7]
```diff
@@ -25,11 +25,13 @@ The `doorbird` implementation allows you to integrate your [DoorBird](https://ww
 There is currently support for the following device types within Home Assistant:
 
 - [Camera](#camera) - View live and historical event based images.
-- [Button](#button) - Enable control of relays and camera night vision.
+- [Button](#button) - Enable control of relays and camera low-light/night vision (IR).
 
 ## Setup
 
-It is recommended to set up a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:
+It is recommended to set up a new & dedicated account on your DoorBird App/web portal for use with Home Assistant. The instructions in this documentation refer specifically to the DoorBird IOS/Android app but most of the instructions can also be performed using the web-based [DoorBird - WebAdmin](https://webadmin.doorbird.com) portal and logging in on your DoorBird admin account.
```
Comment (bdraco) [inline source/_integrations/doorbird.markdown]:
```suggestion
It is recommended to set up a new & dedicated account on your DoorBird App/web portal for use with Home Assistant. The instructions in this document refer specifically to the DoorBird IOS/Android app. Still, most actions can also be performed using the web-based [DoorBird - WebAdmin](https://webadmin.doorbird.com) portal and logging in on your DoorBird admin account.
```

**Review by bdraco — CHANGES_REQUESTED** [review]
Looks good.

Please see small comments above.

**Comment by frenck** [comment]
@rosscullen You 👍 and resolved all the above review comments by bdraco; however, it seems like you have not pushed any changes onto this PR. Did something go wrong?

../Frenck

**Review by frenck — APPROVED** [review]
I've taken the liberty to apply the changes to the PR (so this PR can move forward).

Thanks @rosscullen 👍 

../Frenck

**Pull request merged**